subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/TransformingAsyncArtifactListener.java

```diff
@@ -81,6 +76,6 @@ public void fileAvailable(File file) {
     private <T> void initialSubjectAvailable(T key, TransformationSubject initialSubject, Map<T, TransformationOperation> results, ArtifactTransformDependenciesProvider dependenciesProvider) {
         TransformationOperation operation = new TransformationOperation(transformation, initialSubject, dependenciesProvider);
         results.put(key, operation);
```

Comment:
Can we just use the `result` as value and get rid of `TransformationOperation` altogether?

Reply:
We could, but it's a `List<File>`, and I thought keeping it wrapped and named like this is better. We can revisit this later if we want.

Reply:
The result is a `TransformationSubject`, so it would be already named. Anyway, let's revisit later.